CaImAn/caiman_run.py

```diff
@@ -9,15 +9,15 @@
 
 # Import for CaimAn lib
 import caiman as cm
-from caiman import stop_server
```

Comment:
Shouldn't we stop_server at the end of the processing?

Reply:
still stopping the server in line 225 but doing it like that `cm.stop_server`